web/packages/teleport/package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "pnpm build-wasm && vite",
     "build": "pnpm build-wasm && vite build",
-    "build-wasm": "pnpm --filter=@gravitational/shared build-wasm"
+    "build-wasm": "pnpm --filter=@gravitational/shared build-wasm",
+    "event-reference": "pnpm event-reference:build && pnpm event-reference:gen",
```

Comment:
If you want to make sure that the reference stays up to date as people add or modify events, you'd need to add a CI check similar to the one I added in #57700 for `pnpm process-icons`.